packages/orderbook/README.md

```
@@ -736,6 +750,85 @@ if (result.error) {
+    network: base
+    address: 0x52CEB8eBEf648744fFDDE89F7Bc9C3aC35944775
+    deployment-block: 36667253  # Required!
+    subgraph: base
+```
+
+#### "Missing required field 'description' in gui deployment"
+
+GUI deployments require both a `description` and `fields` array, even if fields are empty or reference scenario bindings.
+
+**Solution:** Ensure your GUI config includes all required fields:
+```yaml
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)